minio/credentials/providers.py

```diff
@@ -453,16 +453,19 @@ class LdapIdentityProvider(Provider):
     """Credential provider using AssumeRoleWithLDAPIdentity API."""
 
     def __init__(
-            self, sts_endpoint, ldap_username, ldap_password, http_client=None,
+            self, sts_endpoint, ldap_username, ldap_password, policy=None,
```

Comment:
[minio-go](https://github.com/minio/minio-go/blob/master/pkg/credentials/sts_ldap_identity.go#L67) doesn't have `policy`. Please discuss minio slack and send fix accordingly.